docs/reference/configuration.md

```diff
@@ -20,29 +20,44 @@ greptime [standalone | frontend | datanode | metasrv]  start -c config/standalon
 Common protocol configurations in `frontend` and `standalone` sub command:
 
 ```toml
+[http_options]
+addr = "127.0.0.1:4000"
+timeout = "30s"
+
 [grpc_options]
-addr = '127.0.0.1:4001'
+addr = "127.0.0.1:4001"
```

Comment:
I noticed that there are still single quotation marks in the latter parts of the file. Would it be better to use a consistent type of quotation mark throughout the entire configuration file?

Reply:
Okay, I will address it.